packages/multichain-network-controller/src/index.ts

```diff
@@ -0,0 +1,23 @@
+export { MultichainNetworkController } from './MultichainNetworkController';
+export { getDefaultMultichainNetworkControllerState } from './constants';
+export type {
+  MultichainNetworkMetadata,
+  SupportedCaipChainId,
+  CommonNetworkConfiguration,
+  NonEvmNetworkConfiguration,
```

Comment:
In bringing back `EvmNetworkConfiguration` did we want to include that type here as well now?

``` suggestion
  NonEvmNetworkConfiguration,
  EvmNetworkConfiguration
```